duo_wordpress.php

```diff
@@ -468,6 +485,13 @@ function duo_update_mu_options() {
         else {
             $result = update_site_option('duo_xmlrpc', 'on');
         }
+        if(isset($_POST['duo_rest'])) {
+            $rest = sanitize_text_field($_POST['duo_rest']);
+            $result = update_site_option('duo_rest', $rest);
```

Comment:
@matgargano Sorry, it's not an issue with WordPress MU -- my point was that the value submitted by the browser could be modified by a bad actor. So I could modify the source on the page to change the value "off" to something else (e.g. `' OR 1=1--`). While this theoretically should not be an issue (you call `sanitize_text_field()`, and hopefully there are no other bugs in the application), it would be ideal if we simply ensure that the value in the DB can only be "off" or "on". If we see that `duo_rest` is set in `$_POST`, we can simply call `$result = update_site_option('duo_rest', "off");`